### PR TITLE
RDM-6544- updated README

### PR DESCRIPTION
**Pull request by akdasari**
### Change description ###
updated the README with latest chart configuration 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,18 @@
 # chart-ccd
 
-NOTE: :warning: version 4.0.0-alpha of the chart-ccd is still not ready to be used. It was released to make testing on HMCTS Demo a faster process
-
 
 This chart is intended for deploying the full CCD product stack.
 Including:
 * data-store-api
```
Comment (mario-paniccia) [inline README.md]: Anand we agreed with Platform Engineering that the services enabled by default will be: Data Store, Def Store, User Profile, Doc Store, Admin Web.
All the others are disabled by default (including the CCD frontends)
Can you update this readme accordingly and create a ticket for that please

Comment (timja) [reply README.md]: does admin web need to be enabled by default?

File: README.md [hunk 2]
```diff
@@ -28,8 +30,24 @@ dependencies:
...
-values.preview.template.yaml
+values.preview.yaml
 ```yaml
+
+# Enable required services as follows
+ccd:
+  testStubsService:
+    enabled: false
+  definitionImporter:
+    enabled: true
```
Comment (timja) [inline README.md]:
```suggestion
    enabled: true
```

File: README.md (hunk 1 above)
Comment (mario-paniccia) [reply README.md]: @timja Without Admin Web it would be difficult for QAs to import definitions

Comment (akdasari) [reply README.md]: added Note about default services

File: ccd/requirements.yaml [hunk 3]
```diff
@@ -10,15 +10,15 @@ dependencies:
     condition: ccd.s2s.enabled
 
   - name: ccd-user-profile-api
-    version: ~1.0.0
+    version: ~1.2.1
```
Comment (mario-paniccia) [inline ccd/requirements.yaml]: please in the future let's avoid mixing up different things in the same ticket. Better to raise a separate one

File: ccd/values.yaml [hunk 4]
```diff
@@ -66,17 +66,17 @@ ccd-case-management-web:
       PRINT_SERVICE_URL_REMOTE: https://return-case-doc-ccd.nonprod.platform.hmcts.net
       CCD_ACTIVITY_URL: https://{{ tpl .Values.global.ccdApiGatewayIngress $}}/activity
       PAYMENTS_URL: https://{{ tpl .Values.global.ccdApiGatewayIngress $ }}/payments
-      DM_URL_REMOTE: "^https?://(?:{{ tpl .Values.global.dmStoreUrl $}}(?::\\d+)?)/documents"
+      DM_URL_REMOTE: "^https?://(?:{{ .Release.Name }}-dm-store(?::\\d+)?)/documents"
```
Comment (mario-paniccia) [inline ccd/values.yaml]: we should have done these changes as part of a separate ticket

File: README.md [hunk 5]
```diff
@@ -1,16 +1,21 @@
...
+Note: Default enabled services with CCD-chart are
+Data Store, Definition Store, User Profile, Document management Store, Api gateway.
+All the others are disabled by default (including the CCD frontends)
 
 This chart is intended for deploying the full CCD product stack.
 Including:
 * data-store-api
 * user-profile-api
 * definition-store-api
+Optional:
```
Comment (mario-paniccia) [inline README.md]: Anand have you checked how this renders? It's messed up:
https://github.com/hmcts/chart-ccd/tree/RDM-6544
please double check yourself before issuing a PR

**Pull request closed without merging**